src/storage-client/src/client.rs

```diff
@@ -146,7 +145,7 @@ impl<T> StorageCommand<T> {
             | InitializationComplete
             | AllowWrites
             | UpdateConfiguration(_)
-            | AllowCompaction(_) => false,
+            | AllowCompaction(_, _) => false,
```

Comment:
Did you know you can use `..` on tuple variants too? Not important at all, but I found this neat when I learned about it :)

Reply:
I did not! Very useful tip, thanks